options/options.h

```diff
@@ -16,6 +16,7 @@ typedef struct mp_vo_opts {
     int ontop_level;
     bool fullscreen;
     bool border;
+    int window_affinity;
```

Comment:
Can move this down where `char *mmcss_profile;` just for consistency.

Reply:
I moved the manual entry near mmcss as well

Reply:
the 2 priority options have a dot behind Windows Only, they will haunt me, no other mention of windows only has that dot, should I remove them with this commit? or can you do a simple doc typo commit after all this?

Reply:
Hah, you can just remove them with this commit. No big deal.

Reply:
Done.